### DISCUSSION TITLE
Farge på dekningsliste er i gammel profil – hva bør vi gjøre med den?

### DISCUSSION THREAD

**Discussion by wkillerud** [fremtind/jokul#2663] [Annet]
Jeg oppdaget underveis i #2662 at én av [listekomponentene](https://jokul.fremtind.no/komponenter/list) våre bruker noen farger som ikke er i [fargeprofilen vår](https://jokul.fremtind.no/profil/farger). Dette er de gamle suksess- og feilfargene til lyst og mørkt tema.

@kennidenni, vet du om dette er noe dere bruker? Hvordan ser disse dekningsikonene ut i lyst og mørkt tema der hvor dere eventuelt ikke bruker denne komponenten?

![Skjermbilde av listekomponenten med ikoner i lyst tema](https://user-images.githubusercontent.com/1223410/150303189-10d8f424-c047-4c00-8fd6-0af23a93bdf9.png)

![Skjermbilde av listekomponenten med ikoner i mørkt tema](https://user-images.githubusercontent.com/1223410/150302832-2d9064ca-d96a-4d10-84a2-e90982682b62.png)

**Comment by kennidenni** [comment]
Vi bruker disse ja, både fra før og i den nye dekningsvelgeren vi holder på å utvikle nå. 
Skade og Liv bruker også disse

**Comment by piofinn** [comment]
Her bør det vel gå an å skrive om til å bruke de nye fargene, og bruke de kraftigere variantene (`suksess--alt` og `feil--alt`) for lyst tema?

https://github.com/fremtind/jokul/blob/main/packages/core/variables/_colors.scss

**Reply by wkillerud** [reply]
Testet nå, og selv `--alt` feiler på kontrastkravet.
